README.md

```diff
@@ -36,11 +36,11 @@ apply(plugin = "build.buf")
 ```
 
 When applied the plugin creates tasks:
-- `bufFormatApply` applies Buf's formatter to protobuf code
-- `bufFormatCheck` validates protobuf code using Buf's formatter
-- `bufLint` lints protobuf code
-- `bufBreaking` checks protobuf against a previous version for backwards-incompatible changes
-- `bufGenerate` generates protobuf code
+- `bufFormatApply` applies Buf's formatter to Protobuf code
```

Comment:
Maybe just link out to the Buf commands these invoke? (`bufFormatApply`: Invokes `buf format`)

Reply:
Let me do a mix of this. I think the intention was to have some description of what the tasks are doing rather than deferring to the Buf docs. I'll add links to the buf documentation for each of the mirrored buf cli commands and i'll call them out in the bullets